crates/ruff_db/src/system/test.rs

```diff
@@ -75,6 +75,13 @@ impl System for TestSystem {
         }
     }
 
+    fn current_directory(&self) -> &SystemPath {
+        match &self.inner {
+            TestFileSystem::Stub(fs) => fs.cwd(),
+            TestFileSystem::Os(fs) => fs.current_directory(),
```

Comment:
What's the reason for the method on one variant using the `cwd` acronym, but the method on the other variant spelling it out as `current_directory()`?

Reply:
No, that's not intentional. Thanks for catching it.